### PR TITLE
Feature/lazy-loading

### PR DESCRIPTION
**Pull request by petrKavulok**
### in this PR
- lazy loading  of all containers in SC-Admin-UI enabled
- removed obsolete code for retrieving usernames in detail screen. This functionality was substituted with` <Credentials /> `component

TODO:

- [ ]  ASAB UI submodule update after merge

### CODE REVIEW

File: src/modules/auth/tenant/TenantDetailContainer.js [hunk 1]
```diff
@@ -38,8 +38,6 @@ function TenantDetailContainer(props) {
 	const [page, setPage] = useState(1);
 	const [filter, setFilter] = useState("");
 	const limit = 10;
-	const [username, setUsername] = useState();
```
Comment (Pe5h4) [inline src/modules/auth/tenant/TenantDetailContainer.js:41]: @petrKavulok Ok, why were those items removed and why it is not in changelog or at least in the description of this PR then? Its very hard to distinguish from reviewers point of view, why this change has been made without any context .

**Review by Pe5h4 — CHANGES_REQUESTED** [review]
@petrKavulok Has it been tested properly? I have a feeling, that some containers was not working properly with lazy loading (not sure if it was resources or roles or some other...)

**Comment by petrKavulok** [comment]
> 

jj, it was. I also remember some of the containers not working, but I have not encountered any funny behaviour while testing

File: src/modules/auth/tenant/TenantDetailContainer.js (hunk 1 above)
Comment (Pe5h4) [inline src/modules/auth/tenant/TenantDetailContainer.js:41]: @petrKavulok PLease add the information about what has been removed and why to the changelog please :)

Comment (petrKavulok) [reply src/modules/auth/tenant/TenantDetailContainer.js:41]: I removed it because I noticed that this functionality was already taken care of by the `<Credentials />` component. I should have made a separate PR for that. Sorry, next time I'll separate it to two PRs

**Review by Pe5h4 — APPROVED** [review]
Looks good
